ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/CveSelections.tsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
+import { Link, generatePath } from 'react-router-dom';
+import { Flex, List, ListItem, Text } from '@patternfly/react-core';
+import { vulnerabilitiesWorkloadCvesPath } from 'routePaths';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+const vulnerabilitiesWorkloadCveSinglePath = `${vulnerabilitiesWorkloadCvesPath}/cves/:cveId`;
 
 export type CveSelectionsProps = {
-    cves: string[];
+    cves: { cve: string; summary: string }[];
 };
 
 function CveSelections({ cves }: CveSelectionsProps) {
     return (
-        <div>
-            {cves.map((cve) => (
-                <p key={cve}>{cve}</p>
+        <List isPlain isBordered>
+            {cves.map(({ cve, summary }) => (
+                <ListItem key={cve}>
+                    <Flex direction={{ default: 'column' }}>
+                        <Text>
+                            <Link
+                                target="_blank"
+                                to={generatePath(vulnerabilitiesWorkloadCveSinglePath, {
+                                    cveId: cve,
```

Comment:
A thought, not sure if it is good: `cveId` here is a possible property key within the cve-summary object.